src/js/components/stock-movement-wizard/modals/AdjustInventoryModal.jsx

```diff
@@ -73,13 +86,47 @@ class AdjustInventoryModal extends Component {
     );
   }
 
-  // temporary disablers
-  /* eslint-disable class-methods-use-this */
-  /* eslint-disable no-unused-vars */
   onSave(values) {
     this.props.showSpinner();
+
+    const url = '/openboxes/api/stockAdjustments';
+
+    return apiClient.post(url, _.map(values.adjustInventory, (adItem) => {
+      const adjustItem = _.find(
+        _.filter(this.state.attr.fieldValue.adItem, listItem => !listItem.initial),
```

Comment:
1. Why you are doing this filtering? 
2. Do you have this.state.attr.fieldValue.adItem?
3. Do those items have 'initial' field?